python/dgl/backend/pytorch/tensor.py

```diff
@@ -277,7 +277,7 @@ def full_1d(length, fill_value, dtype, ctx):
     return th.full((length,), fill_value, dtype=dtype, device=ctx)
 
 def nonzero_1d(input):
-    x = th.nonzero(input).squeeze()
+    x = th.nonzero(input, as_tuple=False).squeeze()
```

Comment:
nit: By default, as_typle is False: https://pytorch.org/docs/master/generated/torch.nonzero.html

Reply:
If we don't specify it explicitly you will get a warning.